WordPress/src/main/java/org/wordpress/android/ui/mysite/SiteItemsBuilder.kt

```diff
@@ -3,20 +3,69 @@ package org.wordpress.android.ui.mysite
 import org.wordpress.android.R
 import org.wordpress.android.fluxc.model.SiteModel
 import org.wordpress.android.ui.mysite.MySiteItem.CategoryHeader
+import org.wordpress.android.ui.mysite.MySiteItem.ListItem
+import org.wordpress.android.ui.utils.ListItemInteraction
 import org.wordpress.android.ui.utils.UiString.UiStringRes
 import javax.inject.Inject
 
 class SiteItemsBuilder
 @Inject constructor(
-    private val siteCategoryItemBuilder: SiteCategoryItemBuilder
+    private val siteCategoryItemBuilder: SiteCategoryItemBuilder,
+    private val siteListItemBuilder: SiteListItemBuilder
 ) {
-    fun buildSiteItems(site: SiteModel): List<MySiteItem> {
+    fun buildSiteItems(site: SiteModel, onClick: ListItemInteraction): List<MySiteItem> {
         val siteItems = mutableListOf<MySiteItem>()
+        siteItems.addItemIfNotNull(siteListItemBuilder.buildPlanItemIfAvailable(site, onClick))
```

Comment:
Maybe we could also rename `addItemIfNotNull` to `addIfNotNull` to match the original `add`.

Reply:
this is gone now